bar-trender-front-end/src/components/withListLoading.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function WithListLoading(Component) {
+  return function WihLoadingComponent({ isLoading, ...props }) {
+    if (!isLoading) return <Component {...props} />;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
    if (!isLoading) {return <Component {...props} />;}
```